tests/unit/records/test_records_wrappers.py

```diff
@@ -101,3 +101,146 @@ def test_literature_record_external_system_identifiers_handles_kekscan():
     result = record.external_system_identifiers
 
     assert expected == result
+
+
+def test_literature_record_publication_information_removes_unrequired_fields():
+    record = LiteratureRecord({
+        'publication_info': [
+            {
+                'artid': '128',
+                'journal_recid': 1214446,
+                'journal_record': {
+                    '$ref': 'http://labs.inspirehep.net/api/journals/1214446'
+                },
+                'journal_issue': '7',
+                'journal_title': 'Astropart.Phys.',
+                'journal_volume': '103',
+                'page_end': '48',
+                'page_start': '41',
+                'year': '2018',
+                'pubinfo_freetext': 'Symmetry 10, 287 (2018)',
+            },
+        ],
+    })
+
+    expected = [
+        {
+            'artid': '128',
+            'journal_issue': '7',
+            'journal_title': 'Astropart.Phys.',
+            'journal_volume': '103',
+            'page_end': '48',
+            'page_start': '41',
+            'year': '2018',
+            'pubinfo_freetext': 'Symmetry 10, 287 (2018)',
+        },
+    ]
+    result = record.publication_information
+
+    assert expected == result
```

Comment:
too many testsss :) you can squash some of those because the don't add any value, for example here you can add ``test_literature_record_publication_information_does_not_keep_objects_without_any_required_fields`` case as well